### PR TITLE
Fix identation in kubeadm api docs

### PR DESCRIPTION
**Pull request by prasadkatti**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Fixes the indentation on the kubeadm config example file

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
Hopefully, this is not a generated file.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/apis/kubeadm/v1beta2/doc.go [hunk 1]
```diff
@@ -233,12 +233,12 @@ limitations under the License.
 // 	scheduler:
 // 	  extraArgs:
 // 	    address: "10.100.0.1"
-// 	extraVolumes:
-// 	- name: "some-volume"
-// 	  hostPath: "/etc/some-path"
-// 	  mountPath: "/etc/some-pod-path"
-// 	  readOnly: false
-// 	  pathType: File
+// 	  extraVolumes:
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/v1beta2/doc.go:236]: please fix this in v1beta1 too:
https://github.com/kubernetes/kubernetes/blob/28d559fcc23de22fe242093310cd79a58abd6f90/cmd/kubeadm/app/apis/kubeadm/v1beta1/doc.go#L241

Comment (prasadkatti) [reply cmd/kubeadm/app/apis/kubeadm/v1beta2/doc.go:236]: done.

**Review by neolit123 — COMMENTED** [review]
thanks
/lgtm
/approve

**Pull request merged**